pennylane/ops/op_math/__init__.py

```diff
@@ -66,14 +66,14 @@
 from .controlled import Controlled, ControlledOp, ctrl
 from .controlled_ops import ControlledQubitUnitary
 from .dot import dot
+from .evolve import Evolve, evolve
```

Comment:
Has `Evolve` been chosen instead of `ParametrizedEvolution`?

Reply:
Maybe a misunderstanding, I just meant that I would want whatever is used inside the `qnode` to be `qml.Evolve`. I see your point with capitalization of functions though, so we can keep `qml.evolve` for the function.
About `ParametrizedEvolution` vs `Evolve` vs `TDEvolve` vs `TDEvolution` I personally dont have a strong preference and would just go with what we have in the UI document, i.e. `ParametrizedEvolution` - sorry for the confusion!

Reply:
Ah! Changing back to `ParametrizedEvolution` then 😄